tests/dummy_server/static/wait_and_tap.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
why not using the wait_and_click.html file? Despite the name (which is only to know what file is what) I'm happy to reuse html files in multiple tests cases if possible

Reply:
I stayed with the separate html files. I think it helps to separate the use cases for ui behavior testing but I can still see the case for reuse. Its a quick change if you'd like to go with the reuse option.